packages/bpk-component-banner-alert/index.js

```diff
@@ -17,8 +17,16 @@
  */
 /* @flow */
 
-import BpkBannerAlert, { ALERT_TYPES } from './src/BpkBannerAlert';
+import BpkBannerAlert from './src/BpkBannerAlert';
+import BpkBannerAlertDismissable from './src/BpkBannerAlertDismissable';
+import BpkBannerAlertExpandable from './src/BpkBannerAlertExpandable';
+import { ALERT_TYPES } from './src/common-types';
 import withBannerAlertState from './src/withBannerAlertState';
 
-export { ALERT_TYPES, withBannerAlertState };
+export {
+  ALERT_TYPES,
+  BpkBannerAlertDismissable,
+  BpkBannerAlertExpandable,
+  withBannerAlertState,
+};
 export default BpkBannerAlert;
```

Comment:
What should be the default export here? Or should we just not have one so the import looks like

```javascript
import { BpkBannerAlert, BpkBannerAlertDismissable, BpkBannerAlertExpandable, ALERT_TYPES } from 'bpk-component-banner-alert';
```

Reply:
I think what you have now make sense, you're exporting by default the main component and the different variants